### PR TITLE
Add a guide for SMS MFA using Vonage

### PR DESCRIPTION
**Pull request by joshcanhelp**
A team review was already completed for this, just needs a docs team review!

cc @aaguiarz 

https://docs-content-staging-pr-8961.herokuapp.com/docs/multifactor-authentication/send-phone-message-hook-vonage

https://docs-content-staging-pr-8961.herokuapp.com/docs/hooks/extensibility-points/send-phone-message

### CODE REVIEW

File: articles/hooks/extensibility-points/send-phone-message.md
Comment (shushen) [inline articles/hooks/extensibility-points/send-phone-message.md]: The contents for customizing SMS providers via hooks are really difficult to discover from MFA docs. Is it possible to add additional links to these provider on pages like: https://auth0.com/docs/mfa/guides/configure-sms, such that customers looking at MFA will be able to find out the other options more easily?

Comment (joshcanhelp) [reply articles/hooks/extensibility-points/send-phone-message.md]: Good call. I'll add some interlinking there.

**Comment by aaguiarz** [comment]
@joshcanhelp The content looks great, and I like the format better than the ones we currently have. However, those were created based on the structure of hooks samples in general, and this new document is not consistent with that format.

@jeffreylees wdyt? Should we change the rest of the documents so it looks like this one, or adjust this one?

**Comment by erjohnson** [comment]
I like the format of this, I'd say altering the others to match this would be great 👯

**Comment by joshcanhelp** [comment]
@aaguiarz @erjohnson - Glad the format is well-liked! Should the format change block this getting merged in or are we OK with the inconsistency while those others are worked on?

Still need to get the interlinking working a bit better, I'll get that in by EOD.

File: articles/multifactor-authentication/send-phone-message-hook-vonage.md
Comment (jeffreylees) [inline articles/multifactor-authentication/send-phone-message-hook-vonage.md:18]: IMO this should just be 

```
[Vonage](https://www.vonage.com/communications-apis/sms/) provides an SMS API which can be used by Auth0 to deliver multi-factor verification via text messages.
```

That says what it is; everything else is a weird sort of ad for Vonage.

And then in the next section, we should remove the screenshot of the Vonage UI. We try hard not to use lots of screenshots of our own UI, and we are trying to (going forward, and as we edit old docs) not screenshot other products' UI at all where possible. Maintenance burden and all.

**Comment by jeffreylees** [comment]
@aaguiarz I added one comment, otherwise I'm good with it. 

I like the new format; it's a little less rigid and flows nicely. Changing the others to be more similar to this should be a project, but not one that blocks releasing this one IMO.

**Comment by joshcanhelp** [comment]
@jeffreylees - Thanks for the review, will get changes up EOD 👍

**Comment by joshcanhelp** [comment]
@aaguiarz @shushen @jeffreylees - Removed my Vonage affiliate link (😆) and updated links in several places:

https://github.com/auth0/docs/pull/8961/commits/4f8cea7740c19ef805744c846bce9ac360848a68

This is ready to merge if you're 👍

**Comment by joshcanhelp** [comment]
@jeffreylees @erjohnson @rkhoriander - Does this need anything else or can we get a final review?

Thank you!

**Comment by rkhoriander** [comment]
Hi, @joshcanhelp . I'll plan to take a look at it this afternoon. I've also labeled it as ready for review, so the concierge should be able to pick it up if they get to it before I do. Thanks!

**Comment by joshcanhelp** [comment]
@rkhoriander - Thank you! I should have looked at the labels ... thanks for adjusting 👍

**Review by aaguiarz — DISMISSED** [review]

**Comment by joshcanhelp** [comment]
@rkhoriander - Sorry to bother but checking in again ... are you able to review this soon?

**Comment by aaguiarz** [comment]
@joshcanhelp @rkhoriander should we rename the new images e.g. /media/articles/multifactor-authentication/09-guide-login-sms-error-message.png, so they have 'vonage' in the name? 

I think the ones that are vonage-specfic should be named different

**Comment by joshcanhelp** [comment]
@aaguiarz - Changed!

@rkhoriander - Anything remaining on your end?

**Review by rkhoriander — APPROVED** [review]

**Pull request merged**
